[2018-08-16T18:02:25.854] [ERROR] errorLog - ReferenceError: item is not defined
    at G:/node/node-service-frame/server/util/jwtAuth.js:8:73
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:8:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:36:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:02:56.517] [ERROR] errorLog - TypeError: Cannot read property 'user' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:8:75
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:8:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:36:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:10:44.805] [ERROR] errorLog - TypeError: Cannot read property 'user' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:8:75
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:8:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:36:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:11:26.984] [ERROR] errorLog - TypeError: Cannot read property 'user' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:10:75
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:10:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:38:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:12:53.790] [ERROR] errorLog - TypeError: Cannot read property 'user' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:12:16
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:10:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:41:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:14:07.805] [ERROR] errorLog - TypeError: Cannot read property 'user' of undefined
    at G:/node/node-service-frame/server/util/jwtAuth.js:10:75
    at Array.reduce (<anonymous>)
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:10:53)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:38:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
[2018-08-16T18:23:26.581] [ERROR] errorLog - ReferenceError: controller is not defined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:10:1)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:103:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
[2018-08-16T19:25:43.970] [ERROR] errorLog - TypeError: Cannot read property 'need_auth' of undefined
    at _callee$ (G:/node/node-service-frame/server/util/jwtAuth.js:14:13)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at jwtAuth (G:\node\node-service-frame\server\util\jwtAuth.js:108:37)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at cors (G:\node\node-service-frame\node_modules\koa2-cors\dist\index.js:91:15)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:7:15)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
